packages/repository/src/relations/has-and-belongs-to-many/has-and-belongs-to-many.decorator.ts

```diff
@@ -0,0 +1,36 @@
+import _ from 'lodash';
+import {
+  DeepPartial,
+  Entity,
+  EntityResolver,
+  HasAndBelongsToManyDefinition,
+  relation,
+  RelationType,
+} from '../..';
+
+export function hasAndBelongsToMany<TH extends Entity, TA extends Entity>(
```

Comment:
Instead of having a decorator for every new relation, have you considered to move the through member to `HasManyDefinition ` so that we can reuse the code and have better UX. Ref: https://github.com/strongloop/loopback-next/pull/5354